src/raw.rs

```diff
@@ -215,9 +253,25 @@ impl RawValue {
     pub fn get(&self) -> &str {
         &self.json
     }
+
+    /// Converts a `Box<RawValue>` into its JSON text as a boxed str without
+    /// copying or allocating.
+    pub fn into_boxed_str(self: Box<Self>) -> Box<str> {
+        unsafe { mem::transmute::<Box<Self>, Box<str>>(self) }
+    }
+    /// Converts a `Box<RawValue>` into its JSON bytes without copying or
+    /// allocating.
+    pub fn into_boxed_bytes(self: Box<Self>) -> Box<[u8]> {
+        self.into_boxed_str().into_boxed_bytes()
+    }
```

Comment:
These seem redundant and niche. The caller can get these representations through APIs on String. Please remove.